designer/client/src/types/definition.ts

```diff
@@ -42,7 +42,6 @@ export interface UIParameter {
     name: string;
     typ: TypingResult;
     editor: $TodoType;
-    validators: $TodoType;
```

Comment:
`validators` are also in `ParameterConfig`. We need to remove them as well

Reply:
Good catch, Removed ✔️